clients/android/app/src/main/java/app/lockbook/loggedin/listfiles/ClickInterface.kt

```diff
@@ -0,0 +1,6 @@
+package app.lockbook.loggedin.listfiles
+
+interface ClickInterface {
+    fun onItemClick(position: Int)
+    fun onLongClick(position: Int) {}
```

Comment:
Why are these different?

Reply:
onLongClick has an empty body because if I do that it is no longer required to be overrided. The reason I need this is because I use this interface for both settings and files, and in settings I only want to register clicks, but in files I want to register both clicks since I handle them differently. Since onLongClick doesn't need to be overrided, I don't have to handle it's click for settings.

Reply:
What's the benefit of re-using the interface? Seems like you could have some unexpected behavior down the line?